subsys/bluetooth/audio/has.c

```diff
@@ -194,9 +202,40 @@ static void security_changed(struct bt_conn *conn, bt_security_t level, enum bt_
 	client = client_get_or_new(conn);
 	if (unlikely(!client)) {
 		BT_ERR("Failed to allocate client");
+		return;
+	}
+
+	/* Notify after reconnection */
+	if (client->ntf_bonded.in_progress) {
+		/* Emit active preset notification */
+		k_work_submit(&active_preset_work);
+
+		/* Emit preset changed notifications */
+		k_work_submit(&client->control_point_work);
```

Comment:
Wrong notification? Should it not be preset_changed, instead of the control point?

Reply:
The Preset Changed notification origins from Control Point. Preset Changes is one of the opcodes.

Reply:
Wait, so if the server, or some other client, changes a value in HAS while the client disconnects, the client can expect a control point notification? That sounds really odd.

Reply:
it's according to HAS specification

Reply:
Yeah, I figured that, but that's just a really odd thing to have in the specification, isn't it? Feels like a gross misuse of the control point concept :(

Reply:
I guess it's too late to change now, right? I'm surprised it if it wasn't raised during IOP of the spec.